prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema

```diff
@@ -234,6 +234,9 @@ elements:
   - name: site_of_care
     csvFields: [{name: Site_of_care}]
 
+  - name: sender_id
+    default: SimpleReport
```

Comment:
Shouldn't this be `simple_report.default` to match what is already in the Covid Result table for Simple Report?

Reply:
I went with SimpleReport to align the format closer to the other senders. Would it be more valuable to keep it the same as it currently is?

Reply:
I think so the current results table has `simple_report.default` results, so it is much simpler to do queries on that table if we don't have to handle the name change for Simple Report.